javascript/javascript1/week2/flight.js

```diff
@@ -0,0 +1,18 @@
+// declaration of variable
+const firstname = 'Hewan';
+const surname = 'Adam';
+
+// declaration of function
+function getFullName(firstname, surname, useFormalName) {
```

Comment:
I recommend you to give `useFormalName` a default value then this line will be
```js
function getFullName(firstname, surname, useFormalName = false) {
```
To read more about that
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters